OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -148,6 +150,11 @@ public void PrerequisitesUnavailable(string key)
 
 		public void PrerequisitesItemHidden(string key) { }
 		public void PrerequisitesItemVisible(string key) { }
+
+		void INotifyCreated.Created(Actor self)
+		{
+			PlayerResources = self.Trait<PlayerResources>();
```

Comment:
Please either use `TraitOrDefault` or declare the requirement on the info.  Right now this hard-dependency isn't exposed to the linter.

Reply:
@IceReaper wanted to make this optional so this will need to be changed anyway.